### DISCUSSION TITLE
Change Logo and Name

### DISCUSSION THREAD

**Discussion by converso-chat** [chatwoot/chatwoot#2701] [Q&A] [answered]
Hello,
How do I change the logo and also the installation name?

Thanks

**Comment by Zenteno** [comment, marked as answer]
I think the easiest way is by modifying the database, in the **installation_configs** table you can modify what you need.

![Captura de pantalla de 2021-07-27 10-32-47](https://user-images.githubusercontent.com/1645609/127172889-567e9ef6-7ef0-4012-8c03-53f1a68c9d5f.png)
You need to modify:

- INSTALLATION_NAME
- BRAND_NAME
- LOGO
- LOGO_THUMBNAIL

After that you need to restart your rails server, and it will take all the changes.

As you can see, the strings are ruby hashed strings, so you have to mantain the format. Maybe they could be changed in the future to a JSON string.

**Reply by WaqqaR** [reply]
I tried these exact steps but im not sure why chatwoot is still showing the old values.

**Reply by sanjayws** [reply]
Try flushing redis cache

#redis-cli
FLUSHALL

**Comment by Guilhermealves20101999** [comment]
o meu não esta dando certo.

**Reply by orlandobcr** [reply]
precisa entrar no shell, escriver "redis-cli" e depois FLUSHALL

**Reply by jose632silva** [reply]
How to access the database?

**Reply by orlandobcr** [reply]
I use pgadmin4 (web admin management tool for postgres),  it may give some trouble installing, as it needs apache and will give error if there is another service using 80 port ).  You can also use Postgres CLI by typing "su - postgres" and then "psql".  Wootchat data base credentials can be found inside ".env" file (inside wootchat installation folder)

**Reply by jose632silva** [reply]
Thanks for the reply, I'll try to do it!

**Comment by uhlhosting** [comment]
There are several dbs, which one contains installation_configs? 

```chatwoot-# \l
                                 List of databases
   Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges   
-----------+----------+----------+------------+------------+-----------------------
 chatwoot  | chatwoot | UTF8     | en_US.utf8 | en_US.utf8 | 
 postgres  | chatwoot | UTF8     | en_US.utf8 | en_US.utf8 | 
 template0 | chatwoot | UTF8     | en_US.utf8 | en_US.utf8 | =c/chatwoot          +
           |          |          |            |            | chatwoot=CTc/chatwoot
 template1 | chatwoot | UTF8     | en_US.utf8 | en_US.utf8 | =c/chatwoot          +
           |          |          |            |            | chatwoot=CTc/chatwoot
(4 rows)
```
